dist/components/top-bar/top-bar.html

```diff
@@ -0,0 +1,105 @@
+<!--
+
+Wikia Style Guide
+Global Top Bar
+
+This component provides a default top bar for Wikia websites and includes handling for logged in user state or a call
+to action for anonymous users.
+-->
+<dom-module id="top-bar">
+  <style>
+    [unresolved] {
+      visibility: hidden;
+    }
+
+    :host {
+      display: block;
+      background-color: #fff;
+      border-bottom: 1px solid #ccc;
+      height: 46px;
+      position: fixed;
+      width: 100%;
+      z-index: 9999999;
+    }
+
+    .logo {
+      fill: #092344;
```

Comment:
Just noticed that the fill color is actually different in mercury than the style guide. Would probably be worth making this consistent at some point, but we also don't have a design for the top nav for mercury, so not a blocker for this.